src/test/java/org/folio/rest/impl/TransactionTest.java

```diff
@@ -4,6 +4,7 @@
 import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
 import static org.folio.rest.impl.StorageTestSuite.storageUrl;
 import static org.folio.rest.impl.TransactionsSummariesTest.ORDER_TRANSACTION_SUMMARIES_ENDPOINT;
+import static org.folio.rest.impl.TransactionsSummariesTest.INVOICE_TRANSACTION_SUMMARIES_ENDPOINT;
```

Comment:
Maybe we should split this test file into separate test files. By default, git doesn't even open it during PR review.

Reply:
split few tests into a different file